ecs-cli/modules/clients/aws/cloudformation/client.go

```diff
@@ -91,6 +91,7 @@ type CloudformationClient interface {
 	CreateStack(string, string, bool, *CfnStackParams) (string, error)
 	WaitUntilCreateComplete(string) error
 	DeleteStack(string) error
+	DescribeStacks(string) (*cloudformation.DescribeStacksOutput, error)
```

Comment:
can we add a test for this in cloudformation/client_test.go ?

Reply:
Since this is just a 3 line function that wraps the cloudformation sdk method of the same name... I don't think its worth it to write a test. I will be adding tests for the other changes in this PR soon tho!

Reply:
Ok, I'll take another look once those are added.